src/main/java/net/minecraftforge/common/ForgeConfig.java

```diff
@@ -127,7 +135,7 @@ public static class Client {
                     .define("useCombinedDepthStencilAttachment", false);
 
             compressLanIPv6Addresses = builder
-                    .comment("When enabled, Forge will convert discovered 'Open to LAN' IPv6 addresses to their more compact, compressed representation")
+                    .comment("[DEPRECATED] [NOW IN COMMON CONFIG] When enabled, Forge will convert discovered 'Open to LAN' IPv6 addresses to their more compact, compressed representation")
```

Comment:
This doesn't seem to be in the common config currently but does seem to be deprecated/unused? @PaintNinja Can you explain should this comment just have the deprecated bit and not the now in common config part?

Reply:
Spoke with Lex, it's now always-on with no option to turn it off. The original PR intended on merging the compression for logging and LAN discovery into a single config option.

The reason for there being an option to turn off IP compression for LAN discovery was a "just in case" scenario where I wanted to give people something to try if they came to me with any problems and it turned out to be IPv6 LAN working *except* when using compressed addresses. It's been a long time and that's never been an issue for anyone, so Lex changed it to be always-on.

So basically: debugging option deemed unnecessary after a long time of unuse -> new unified option for logging and debug lan discovery got removed from the PR before merge -> old config option remains in case someone programmatically reads that setting and expects it to exist but no longer has any effect.

Rename it to `"[DEPRECATED] [NO EFFECT]"`

Reply:
Wouldn't we just remove it then?

Reply:
> old config option remains in case someone programmatically reads that setting and expects it to exist but no longer has any effect